src/blocks/ImageTiles/ImageTilesData.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Reflect the changes for the react components

Reply:
Done, changes made in the react component

Reply:
👍